### ISSUE TITLE
Feature Request: Ability to build documentation outside of source tree

### ISSUE THREAD

**Issue by LeakyBucket** [rust-lang/mdBook#1095]
I have a situation where I would like to build documentation that is maintainable both from the perspective of the repository but also potentially from the perspective of the mdbook.

In my situation I also have multiple repositories and would like to centralize the documentation for all of them.

I had hoped/expected that I could use git submodules and give the relative path to the markdown files already in the repo to build the book.  Unfortunately that does not work as the `html` is generated in the same directory as the `md` file rather than being generated in the `book` directory.

tldr:  Is there a way to reference `md` files from other projects/repositories and build a self-contained `mdbook` from those files?  One that doesn't need those repos to be checked out and in the same location in order to be read.

**Comment by marco-m-pix4d** [comment]
Hello, this looks like a duplicate of #1347 ? In that issue there are also some workarounds.
